src/features/campaigns/components/ActivityList/items/TaskListItem.tsx

```diff
@@ -12,9 +12,8 @@ interface TaskListItemProps {
 }
 
 const TaskListItem = ({ orgId, taskId }: TaskListItemProps) => {
-  const model = useModel((env) => new TaskModel(env, orgId, taskId));
-  const task = model.getTask().data;
-  const stats = model.getTaskStats().data;
+  const { data: task } = useTask(orgId, taskId);
+  const { data: stats, isLoading: statsLoading } = useTaskStats(orgId, taskId);
```

Comment:
I just wonder why you chose to do this instead of just {data, isLoading} ?

Reply:
task, stats and statsLoading is used here and there in `TaskListItem`, and I didn't want to make so many changes to other codes so I unpacked like that but I can change it. No problem!

Reply:
You're right, I see now that the renaming of the variables is necessary, well done 💯